sweep/fee_bumper.go

```diff
@@ -1415,6 +1420,38 @@ func (t *TxPublisher) getSpentInputs(
 				"%v", op, heightHint)
 		}
 
+		// Check whether the input has been spent or not.
+		utxo, err := t.cfg.ChainIO.GetUtxo(
```

Comment:
Hmm, will this also populate the spend cache for neutrino backends? Otherwise, this can be a very expensive filter rescan depending on how far back they are. 

In other words, this'll block for neutrno backends. Would need to check for behavior with backends that have the txindex off.

Reply:
What about just moving back to the spend channel/goroutine? That way it's always active, always watching, and we can handle the notification async when needed. 

It would allow us to remove all these other default select cases for spend ntfns. I recall I pointed out a possibility of missed events when this change was originally added.

Reply:
> Hmm, will this also populate the spend cache for neutrino backends? Otherwise, this can be a very expensive filter rescan depending on how far back they are.

FWIW this is already used in `RegisterSpendNtfn` implemented in neutrino,
https://github.com/lightningnetwork/lnd/blob/4389067989f27dc315d491cc5c6c24a434b38891/chainntnfs/neutrinonotify/neutrino.go#L864

> What about just moving back to the spend channel/goroutine? That way it's always active, always watching, and we can handle the notification async when needed.

Can try that route, meanwhile there's #10117 that fixes this issue using an alternative approach. I will see if it's possible to make a new sync method when implementing SQL into btcwallet.